sound/soc/sof/ipc4-pcm.c

```diff
@@ -835,5 +835,6 @@ const struct sof_ipc_pcm_ops ipc4_pcm_ops = {
 	.dai_link_fixup = sof_ipc4_pcm_dai_link_fixup,
 	.pcm_setup = sof_ipc4_pcm_setup,
 	.pcm_free = sof_ipc4_pcm_free,
-	.delay = sof_ipc4_pcm_delay
```

Comment:
commit message:

then the host sets the RUN bit during PCM trigger.
FW programs the DMA when the SET_PIPELINE_STATE IPC is sent during PCM trigger

It would be good to avoid "PCM trigger" for two different cases. We have
snd_sof_pcm_platform_trigger(sdev, substream, cmd); and 	pcm_ops->trigger(component, substream, cmd);
and it'd be good to refer more clearly to each. "PCM trigger" applies to both.